site/src/sphinx/client-circuit-breaker.rst

```diff
@@ -111,40 +111,81 @@ is raised or the status is ``5xx``, succeeds when the status is ``2xx`` and igno
 
 .. code-block:: java
 
+    import com.linecorp.armeria.client.ClientRequestContext;
+    import com.linecorp.armeria.client.UnprocessedRequestException;
+    import com.linecorp.armeria.common.HttpStatus;
     import com.linecorp.armeria.common.HttpStatusClass;
 
     final CircuitBreakerStrategy<HttpResponse> myStrategy = new CircuitBreakerStrategy<HttpResponse>() {
 
         @Override
-        public CompletionStage<Boolean> shouldReportAsSuccess(HttpResponse response) {
-            return response.aggregate().handle((res, cause) -> {
-                if (cause != null) { // A failure if an Exception is raised.
-                    return false;
+        public CompletionStage<Boolean> shouldReportAsSuccess(ClientRequestContext ctx,
+                                                              @Nullable HttpResponse response,
+                                                              @Nullable Throwable cause) {
+            if (cause != null && !(cause instanceof UnprocessedRequestException)) {
+                // A failure if an Exception is raised.
+                return CompletableFuture.completedFuture(false);
```

Comment:
How about adding `return CompletableFuture.completedFuture(null)` for `UnprocessedRequestException` around here? We could also add some comments about why `null` has to be returned on `UnprocessedRequestException`.